Automattic/wp-calypso
# SiteAddressChanger: Remove devdocs example

### Summary

As new behaviour and new actions have been added to the Site Address Changer feature, the devdocs has fallen behind and no longer works - this is a mix of errors that result from missing props/actions and behavioural changes that now cause the example site id given in the devdocs to show the `Only the site owner can edit this domain name.` message.

Because the component is no longer reliably demo-able this PR aims to remove it from the DevDocs completely.

### Testing

- Go to http://calypso.localhost:3000/devdocs/blocks/simple-site-rename-form
  - You should no longer see an entry for the component
- Go to http://calypso.localhost:3000/devdocs/blocks
  - You should no longer see an entry for the component in the list of blocks
